src/Symfony/Bundle/FrameworkBundle/Console/Descriptor/JsonDescriptor.php

```diff
@@ -106,6 +106,9 @@ protected function describeContainerServices(ContainerBuilder $builder, array $o
             if ($service instanceof Alias) {
                 $data['aliases'][$serviceId] = $this->getContainerAliasData($service);
             } elseif ($service instanceof Definition) {
+                if (!$showHidden && $service->hasTag('container.excluded')) {
```

Comment:
Hidden services are true services. Mixing them with excluded services would give a false impression that those are available for use. Let's just skip them in all cases

Reply:
I agree. Rule for hidden services already exists in [102](https://github.com/symfony/symfony/blob/48a333caf283e005fd49a7824a90533f430121e3/src/Symfony/Bundle/FrameworkBundle/Console/Descriptor/JsonDescriptor.php#L102) line, so I deleted this. Is that better?

Reply:
👍